readme.md

```diff
@@ -33,3 +33,6 @@ Return a newline-separated list of ids which link to the provided `:id`..
 Return a newline-separated list of JSON documents for each observation in the
 database.
 
+# data
+
+Data is stored in `path.join(require('ospath').data(), 'mapfilter-osm-p2p')`.
```

Comment:
Could you mention that this is `/Users/USERNAME/Library/Application Support` on macOS and `/home/USERNAME/.config` on Linux? (I'm not sure what Windows path this translates to.)